packages/contracts/contracts/Passport.sol

```diff
@@ -8,19 +8,72 @@ import "@openzeppelin/contracts/access/Ownable.sol";
  * The Passport contract does this and that...
  */
 contract Passport is ERC721, Ownable {
+  bytes4 private constant _INTERFACE_ID_ERC2981 = 0x2a55205a;
+
+  struct RoyaltyInfo {
+      address recipient;
+      uint24 amount;
+  }
```

Comment:
Is this struct necessary? Isn't `recipient` always the `owner` and `amount` always dependent on `royaltyPcnt`?

If it's redundant, we should nix it as extra memory === extra gas

Reply:
Yes, we can nix it.

Reply:
oh I suppose the owner could specify a new recipient for the royalty. That part is not necessarily redundant